app/js/newsfeed.js

```diff
@@ -119,7 +119,7 @@ module.exports = {};
    * @returns {Boolean} Always returns true.
    */
   function displayNews(posts) {
-    loadingDots.parentElement.removeChild(loadingDots);
+    loadingDots.remove();
```

Comment:
`.remove()` is a long-needed shortcut to what you had. See also: https://developer.mozilla.org/en-US/docs/Web/API/ChildNode/remove